src/components/__tests__/CheatMode.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import { fireEvent, waitFor } from '@testing-library/react-native'
+import React from 'react'
+
+import { SIMPLE_RESULTS } from '../../constants/data'
+import labels from '../../constants/labels.json'
+import { getDevMode } from '../../services/AsyncStorage'
+import render from '../../testing/render'
+import CheatMode from '../CheatMode'
+
+jest.mock('../../services/AsyncStorage', () => ({
+  getDevMode: jest.fn(),
+}))
+
+const cheatFunc = jest.fn()
```

Comment:
We don't use type suffixes, functions should just always be or contain a verb:
```suggestion
const cheat = jest.fn()
```